### ISSUE TITLE
Drag&drop support between two grids

### ISSUE THREAD

**Issue by kv92** [bencripps/react-redux-grid#150]
Hi,
Thanks for a fantastic grid/treegrid component!

As far as I understand DnD of nodes between different grids is not currently supported? Do you have plans on adding such feature, maybe you can share your thought on how difficult it would be to create it?

Thanks,
Vasiliy

**Comment by bencripps** [comment]
I don't think it would be that difficult since the full data object comes in the drop event, I'm not sure if the `stateKey` does, however. That would be easy enough to add. I will mark this as help-wanted and get to it as soon as I can. As always, pull requests are very welcome!

**Comment by baloraki** [comment]
Hi Ben,
Thanks for the nice grid module.
Could you already find time for the feature?
